airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -164,6 +178,7 @@ def __init__(
         worker_disk_type: str = "pd-standard",
         worker_disk_size: int = 1024,
         num_preemptible_workers: int = 0,
+        preemptibility: str = " ",
```

Comment:
Is there a typo here?
```suggestion
        preemptibility: str = "",
```
And why not setting it to the default value `preemptible`?

Reply:
Yes, you are right here, i have updated my code :)